tests/integration/telemetry/util/util.go

```diff
@@ -15,19 +15,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package telemetry
+package util
```

Comment:
i'm curious: why refactor this package (or part of the package)?  `telemetry` is a more descriptive name.  can this be combined with the `common` stuff to make a single `telemetry` package?